Tests/LiquidProjections.NHibernate.Specs/packages.config

```diff
@@ -1,14 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>

```

Comment:
Would be better to update packages in separate commit(s)

Reply:
Yeah, I agree. But it was too late.